lib/parser-task.js

```diff
@@ -25,7 +25,11 @@ module.exports = function (path) {
     window = undefined;
     /* eslint-enable no-native-reassign, no-undef */
 
-    require(path + '/Gruntfile.js')(grunt);
+    if (fs.existsSync(path + '/Gruntfile.coffee')) {
+      require(path + '/Gruntfile.coffee')(grunt);
+    } else {
+      require(path + '/Gruntfile.js')(grunt);
+    }
```

Comment:
`fs` is not defined. You must require it. Also, are you sure it is possible to `require` a `coffee` file?